src/utils/RealtimeFeedUtils.js

```diff
@@ -172,8 +87,33 @@ function getDelayInformation(stopID: String, tripID: String, rtf: Object): ?Obje
   };
 }
 
+function getVehicleInformation(
+  routeID: String,
+  tripID: String,
+  vehicles: Object,
+): ?Array<Object> {
+  // vehicles param ensures the vehicle tracking information doesn't update in
+  // the middle of execution
+  if (!routeID
+    || !tripID
+    || !vehicles
+    || vehicles === {}) {
+    LogUtils.log({
+      category: 'getVehicleInformation NULL',
+      routeID,
+      tripID,
+    });
+    return null;
+  }
+  return Object.values(vehicles).filter(
+    v => v.routeID === routeID && v.tripID === tripID,
```

Comment:
hm, isn't it convention to have `v =>` on the previous line to make this easier to read?

Reply:
ohh i didnt know that, gotcha!